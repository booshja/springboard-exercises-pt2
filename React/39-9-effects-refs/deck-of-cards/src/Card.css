.Card {
    position: absolute;
    z-index: 100;
    left: 0;
    right: 0;
    margin: auto;
    background: transparent;
    border-radius: 13px;
}
