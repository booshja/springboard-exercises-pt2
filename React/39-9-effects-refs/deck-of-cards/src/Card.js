import React from "react";
import "./Card.css";

const Card = ({ image, name }) => {
    return <img src={image} alt={name} className="Card" />;
};

export default Card;
