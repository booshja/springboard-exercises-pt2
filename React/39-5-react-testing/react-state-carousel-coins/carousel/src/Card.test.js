import React from "react";
import { render } from "@testing-library/react";
import Card from "./Card";

it("renders without crashing", () => {
    render(
        <Card
            caption="This is a test"
            src="source.jpg"
            currNum="1"
            totalNum="2"
        />
    );
});

it("should match snapshot", () => {
    const { asFragment } = render(
        <Card
            caption="This is a test"
            src="source.jpg"
            currNum="1"
            totalNum="2"
        />
    );
    expect(asFragment()).toMatchSnapshot();
});
