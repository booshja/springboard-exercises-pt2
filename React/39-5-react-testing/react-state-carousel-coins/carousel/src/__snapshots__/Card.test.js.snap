// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      This is a test
    </h4>
    <img
      alt="This is a test"
      class="Card-image"
      src="source.jpg"
    />
    <small
      class="Card-small"
    >
      Image 1 of 2.
    </small>
  </div>
</DocumentFragment>
`;
